packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -154,4 +155,78 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     (this[$sourceObject] as GLTFMaterial).emissiveFactor = rgb;
     this[$onUpdate]();
   }
+
+  setAlphaCutoff(cutoff: number): void {
+    cutoff = Math.min(1.0, cutoff);
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      material.alphaTest = cutoff;
+      material.needsUpdate = true;
+    }
+    (this[$sourceObject] as GLTFMaterial).alphaCutoff = cutoff;
+    this[$onUpdate]();
+  }
+
+  getAlphaCutoff(): number {
+    return (this[$sourceObject] as GLTFMaterial).alphaCutoff ??
+        this[$correlatedObjects]!.values().next().value.alphaTest;
+  }
+
+  setDoubleSided(doubleSided: boolean): void {
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      // When double-sided is disabled gltf spec dictates that Back-Face culling
+      // must be disabled, in three.js parlance that would mean FrontSide
+      // rendering only.
+      // https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#double-sided
+      material.side = doubleSided ? DoubleSide : FrontSide;
+      material.needsUpdate = true;
+    }
+    (this[$sourceObject] as GLTFMaterial).doubleSided = doubleSided;
+    this[$onUpdate]();
+  }
+
+  getDoubleSided(): boolean {
+    return (this[$sourceObject] as GLTFMaterial).doubleSided ??
+        this[$correlatedObjects]!.values().next().value.side === DoubleSide;
+  }
+
+  setAlphaMode(alphaMode: AlphaMode): void {
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      material.blending = NormalBlending;
```

Comment:
Is this never set to another value?

Reply:
No, I had noticed that materials that should be 'opaque' (according to the file spec) are actually set to 'blend' on import, my assumption was that this was intentional, because 'blend' In combination with cutoff hack (set to -0.5) results in an opaque material. However, no that you mention it, this would probably interfere with render performance, on large scenes anyway, where sorting front-to-back actually makes a difference.

Reply:
From the docs it sounds as though Three uses the `transparent` flag to do sorting.
https://threejs.org/docs/#api/en/materials/Material.transparent